clients/typescript/test/satellite/process.test.ts

```diff
@@ -1181,10 +1181,9 @@ test('handling connectivity state change stops queueing operations', async (t) =
 
   satellite._connectivityStateChange('connected')
```

Comment:
This should fix the test: what triggers the reconnect is the connections becoming available

```suggestion
  satellite._connectivityStateChange('available')
```

Reply:
Done!